components/mosart/src/riverroute/RtmHistFlds.F90

```diff
@@ -164,15 +164,15 @@ subroutine RtmHistFldsInit()
          avgflag='A', long_name='WRM supply provided ', &
          ptr_rof=StorWater%Supply, default='active')                                                                                                                              
```

Comment:
I worry that the units change would confuse users because time step is not easily found in outputs.